<launch>
    <!-- Launches downsampler-->
    <node pkg="pcl_n_lidar" type="camera_pcl_downsampler_node" name="CamDownsample" output="screen">
        <rosparam file="$(find pcl_n_lidar)/params/camera_downsample_params.yaml" command="load" />
    </node>
</launch>